taskcluster/ci/config.yml

```diff
@@ -123,6 +123,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/addons-search-detection
             default-ref: main
             type: git
+        searchengineusagestudy:
+            name: "Rally Search Engine Usage study extension"
+            project-regex: searchengineusagestudys$
+            default-repository: https://github.com/mozilla-rally/search-engine-usage-study
```

Comment:
This will need to be `git@github.com:mozilla-rally-search-engine-usage-study` for the private repo or the build will hang.